data/presets/shop/hairdresser/barber.json

```diff
@@ -0,0 +1,42 @@
+{
+    "icon": "temaki-beauty_salon",
```

Comment:
optimally, this would have a dedicated icon, but I couldn't find a suitable one in the current icon sets. Maybe something like a [razor](https://www.svgrepo.com/svg/300806/razor-barber) could work…

Reply:
Or a [barber pole](https://stock.adobe.com/292296536)?